stories/topics/proc-create-credentials-cmd-generator.adoc

```diff
@@ -0,0 +1,59 @@
+[id="prod-create-credentials-cmd-generator"]
+
+= Create credentials for the Command Generator
+
+It is required to have an AWS credentials file for all day-2 operations, like adding and removing Extension Nodes.
+
+The file must follow the format of the AWS credentials file for "Long-term credentials", which includes the `aws_access_key_id` and `aws_secret_access_key` variables. For the official AWS documentation, read link:https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html[AWS Command Line Interface User Guide].
+
+An example of the credentials file format is:
+
+[NOTE]
+====
+It is required to use the `default` profile name in the credentials file. Any other profile name will not be recognized by the Command Generator. For more information, read xref:con-tech-note-cmd-generator-aws-credentials[Command generator - AWS Credentials File] technical note.
+====
+
+[literal, options="nowrap" subs="+attributes"]
+----
+[default]
+aws_access_key_id=AKIAIOSFODNN7EXAMPLE
+aws_secret_access_key=wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
+----
+
+After the credentials file is created, you can use it in the command generator by passing the file path to the `-c` option. Example:
+
+/home/user/project/extra_vars.yaml:
```

Comment:
Do you simply enter the path? Or is there an associated command?

Reply:
I did some changes in the format.